tests/test_opds.py

```diff
@@ -1102,12 +1107,20 @@ def test_groups_feed(self):
         )
 
         annotator = TestAnnotatorWithGroup()
+        private = object()
         cached_groups = AcquisitionFeed.groups(
             self._db, "test", self._url, self.fantasy, annotator,
             max_age=0, search_engine=search_engine,
-            search_debug=True
+            search_debug=True, private=private
         )
-        parsed = feedparser.parse(cached_groups)
+
+        # The result is an OPDSFeedResponse object. The 'private'
+        # argument, unused by groups(), was passed along into the
+        # constructor.
+        assert isinstance(cached_groups, OPDSFeedResponse)
+        eq_(private, cached_groups.private)
+
+        parsed = feedparser.parse(cached_groups.data)
```

Comment:
`.data` since `cached_groups` is the Response and not the content, right?

Reply:
Right.